code/modules/events/_event.dm

```diff
@@ -93,19 +93,38 @@
 		SSblackbox.record_feedback("tally", "event_admin_cancelled", 1, typepath)
 
 /datum/round_event_control/proc/runEvent(random = FALSE)
+	//We clear our signals first
+	UnregisterSignal(SSdcs, COMSIG_GLOB_RANDOM_EVENT)
 	var/datum/round_event/E = new typepath()
 	E.current_players = get_active_player_count(alive_check = 1, afk_check = 1, human_check = 1)
 	E.control = src
-	SSblackbox.record_feedback("tally", "event_ran", 1, "[E]")
 	occurrences++
 
 	testing("[time2text(world.time, "hh:mm:ss")] [E.type]")
+	triggering = TRUE
+
+	if (alert_observers)
+		message_admins("Random Event triggering in [RANDOM_EVENT_ADMIN_INTERVENTION_TIME] seconds: [name] (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")
```

Comment:
I don't see where `cancel` is used here.

Reply:
its used in topic ref i had tested it out in a local server and it worked to make sure im not mixing it up with prerunevent I used a different message in message_admins

Reply:
Ah right